File: src/main/kotlin/me/huanmeng/opensource/bukkit/util/MathUtil.java
package me.huanmeng.opensource.bukkit.util;

import java.util.ArrayList;
import java.util.List;

/**
 * 2023/3/17<br>
 * Gui<br>
 *
 * @author huanmeng_qwq
 */
public class MathUtil {
    private MathUtil() {
        throw new UnsupportedOperationException();
    }

    public static int[] range(int min, int max) {
        List<Integer> list = new ArrayList<>();
        for (int i = min; i <= max; i++) {
            list.add(i);
        }
        return list.stream().mapToInt(Integer::intValue).toArray();
    }
}

File: src/main/kotlin/me/huanmeng/opensource/bukkit/util/MathUtil.kt
package me.huanmeng.opensource.bukkit.util

/**
 * 2023/3/17<br></br>
 * Gui<br></br>
 *
 * @author huanmeng_qwq
 */
object MathUtil {
    /**
     * 获取一个范围内的随机数
     *
     * @param min 最小值
     * @param max 最大值
     * @return 数组
     */
    @JvmStatic
    fun range(min: Int, max: Int): IntArray {
        return min.rangeTo(max).toList().toIntArray()
    }
}
